CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ Code changes are welcome and should follow the guidelines below.
 * Fork the repository on GitHub.
 * Add tests for your new code ensuring that you have 100% code coverage (we can help you reach 100% but will not merge without it).
     * Run `npm test` to generate a report of the test coverage
-* [Pull requests](http://help.github.com/send-pull-requests/) should be made to the [master branch](https://github.com/Lob/Lob-node/tree/master).
+* [Pull requests](https://help.github.com/articles/about-pull-requests/) should be made to the [master branch](https://github.com/Lob/Lob-node/tree/master).
```

Comment:
while you're at it, can you change `https://github.com/Lob/Lob-node/tree/master` to `https://github.com/lob/lob-node/tree/master` (just lowercase it)